src/services/completions.ts

```diff
@@ -3004,6 +3004,7 @@ namespace ts.Completions {
         function tryGetObjectTypeLiteralInTypeArgumentCompletionSymbols(): GlobalsSearch | undefined {
             const typeLiteralNode = tryGetTypeLiteralNode(contextToken);
             if (!typeLiteralNode) return GlobalsSearch.Continue;
+            if (typeLiteralNode.flags & (NodeFlags.JSDoc & ~NodeFlags.JavaScriptFile)) return GlobalsSearch.Fail;

```

Comment:
This is just `NodeFlags.JSDoc` - same as https://github.com/microsoft/TypeScript/pull/50757#discussion_r986526874 and https://github.com/microsoft/TypeScript/pull/51055#discussion_r987202292, right?